TombEngine/Game/effects/tomb4fx.h

```diff
@@ -171,6 +171,14 @@ struct BLOOD_STRUCT
 	byte pad;
 };
 
+enum ShockwaveDamage
+{
+	none,
```

Comment:
Enum members should be PascalCase too (first letter of every word is capitalized).
Also you must use `enum class` to prevent implicit casts to int/short/etc numerical type.

Reply:
removed enum ShockwaveDamage